iothub_client/tests/iothub_client_properties_ut/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 #Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 #this is CMakeLists.txt for version
-cmake_minimum_required(VERSION 2.8.11)
+cmake_minimum_required(VERSION 3.5)
```

Comment:
Not sure if we should do this in the LTS branch. Do we really need the new version right now?

Reply:
I'm not sure if cmake would give a warning for the version being too old, but since the last LTS was released just 3 months ago I would say it's fine to update it now. Don't you think? The code is open, customers can tweak these as well.